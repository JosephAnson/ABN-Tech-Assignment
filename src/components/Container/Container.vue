<template>
  <div class="container max-lg:max-w-full">
    <slot></slot>
  </div>
</template>
