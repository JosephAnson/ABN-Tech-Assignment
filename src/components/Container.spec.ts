import { render } from "@testing-library/vue"
import { describe, expect, test } from "vitest"
import Container from "./Container.vue"

describe("Container", () => {
  test("should render container", () => {
    const { html } = render(Container, {
      slots: {
        default: "Contained Content",
      },
    })
    expect(html()).toMatchSnapshot()
  })
})
