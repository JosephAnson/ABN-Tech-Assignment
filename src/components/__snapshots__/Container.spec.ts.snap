// Vitest Snapshot v1

exports[`Container > should render container 1`] = `"<div class=\\"container max-lg:max-w-full\\">Contained Content</div>"`;
